utopia-remix/app/env.server.ts

```diff
@@ -19,12 +19,13 @@ export const ServerEnvironment = {
   AUTH0_CLIENT_ID: optionalEnv('AUTH0_CLIENT_ID', '<AUTH0_CLIENT_ID>'),
   AUTH0_REDIRECT_URI: optionalEnv('AUTH0_REDIRECT_URI', '<AUTH0_REDIRECT_URI>'),
   // FGA Credentials
-  FGA_STORE_ID: optionalEnv('FGA_STORE_ID', '<FGA_STORE_ID>'),
-  FGA_CLIENT_ID: optionalEnv('FGA_CLIENT_ID', '<FGA_CLIENT_ID>'),
-  FGA_SECRET: optionalEnv('FGA_SECRET', '<FGA_SECRET>'),
-  FGA_API_HOST: optionalEnv('FGA_API_HOST', '<FGA_API_HOST>'),
-  FGA_API_TOKEN_ISSUER: optionalEnv('FGA_API_TOKEN_ISSUER', '<FGA_API_TOKEN_ISSUER>'),
-  FGA_API_AUDIENCE: optionalEnv('FGA_API_AUDIENCE', '<FGA_API_AUDIENCE>'),
+  FGA_STORE_ID: optionalEnv('FGA_STORE_ID', ''),
```

Comment:
Seems a little strange to default the value that would come from an environment variable to an empty string.

Reply:
the idea here is that in most of the scenarios we dont need this value in local env file